src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+        /// Values in the range [0-100], default value is 75 [percent]. Other values might produce unexpected side-effects.
+        /// </summary>
+        public float GaugeLabelsFontPct
+        {
+            get => _GaugeLabelsFontPct;
+            set => _GaugeLabelsFontPct = (value > 100 ? 100 : (value < 0 ? 0 : value));
+        }
+        private float _GaugeLabelsFontPct = 75f;
+
+        /// <summary>
+        /// <see langword="Font"/> used for labeling values on the plot
+        /// </summary>
+        public Drawing.Font Font { get; set; } = new();
+
+        /// <summary>
+        /// <see langword="True"/> if value labels are shown inside the gauges.
+        /// Size of the text is set by <see cref="GaugeLabelsFontPct"/> and color by <see cref="GaugeLabelsColor"/>.
+        /// </summary>
+        public bool ShowGaugeValues { get; set; } = true;
+
+        public System.Drawing.Drawing2D.LineCap EndCap { get; set; } = System.Drawing.Drawing2D.LineCap.Triangle;
+
+        public System.Drawing.Drawing2D.LineCap StartCap { get; set; } = System.Drawing.Drawing2D.LineCap.Round;
+
+        /// <summary>
+        /// Controls rendering style of the concentric circles (ticks) of the web
+        /// </summary>
+        //public RadarAxis AxisType { get; set; } = RadarAxis.None;
+
+        // These 3 properties are needed as part of IPlottable
+        public bool IsVisible { get; set; } = true;
+        public int XAxisIndex { get; set; } = 0;
+        public int YAxisIndex { get; set; } = 0;
+
+        #endregion Properties & fields
+
+        /// <summary>
+        /// Initializes the plot instance.
+        /// </summary>
+        /// <param name="values">Array of values to be plotted as gauges.</param>
+        /// <param name="lineColors">Array colors for the gauges.</param>
+        public RadialGaugePlot(double[] values, Color[] lineColors)
+        {
+            GaugeColors = lineColors;
+            Update(values);
+        }
+
+        public override string ToString() =>
+            $"RadialGauge with {DataRaw.Length} points.";
+
+        /// <summary>
+        /// Replace the data values with new ones. This passed data is copied and stored in <see cref="DataRaw"/>.
+        /// Implicitly calls the <see cref="ComputeAngularData"/> routine
+        /// </summary>
+        /// <param name="values">Array of values to be plotted as gauges.</param>
+        public void Update(double[] values)
+        {
+            DataRaw = new double[values.Length];
+            Array.Copy(values, 0, DataRaw, 0, values.Length);
+
+            // Compute MaxScale and MinScale values and transform DataRaw into DataAngular
+            ComputeMaxMin();
+            ComputeAngularData();
+        }
+
+        /// <summary>
+        /// Returns the original data in case it should be needed.
+        /// </summary>
+        /// <returns>Original data vector</returns>
+        public double[] GetData() => DataRaw;
```

Comment:
I think you have a getter for this already. If not, I would just change it to something like this:

```cs
public double DataRaw { public get; private set; }
```

Reply:
That's true.
https://github.com/ScottPlot/ScottPlot/blob/d7f90e1346e91c60c85a45e2d2f75b36302bd19a/src/ScottPlot/Plottable/RadialGaugePlot.cs#L44-L48
However, I get [compiler error CS0274](https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0274?f1url=%3FappId%3Droslyn%26k%3Dk(CS0274)) when trying the suggested code.
I'm at a loss here.

Reply:
I'm actually going to make this `private` to keep the API small. I will expect the end user to be responsible for maintaining their own data 😈 

@arthurits is correct that @bclehmann's code example throws a compile error. I couldn't figure out why at first, but eventually figured it out:

```cs
// this causes compile error CS0274
public double[] DataRaw { public get; private set; }
```

```cs
// this works
public double[] DataRaw { get; private set; }
```